recipes/plaster_pastedeploy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plaster_pastedeploy" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c231130cb86ae414084008fe1d1797db7e61dc5eaafb5e755de21387c27c6fae
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
```

Comment:
Sorry, scratch that. We only need to define entry points if they are `console_scripts`. Other entry point types will have their metadata properly installed by `pip`. Sorry 😅

Reply:
Done